src/main/java/com/moddy/server/controller/model/ModelController.java

```diff
@@ -40,4 +41,19 @@ public SuccessResponse<ModelMainResponse> getModelMainInfo(
         return SuccessResponse.success(SuccessCode.FIND_MODEL_MAIN_INFO_SUCCESS, modelService.getModelMainInfo(userId, page, size));
     }
 
+    @Operation(summary = "[JWT] 제안서 상세보기 뷰 조회", description = "제안서 상세보기 API입니다.")
+    @ApiResponses({
+            @ApiResponse(responseCode = "200", description = "모델 메인뷰 조회 성공", content = @Content(schema = @Schema(implementation = ModelMainResponse.class))),
+            @ApiResponse(responseCode = "401", description = "인증 오류 입니다.", content = @Content(schema = @Schema(implementation = ErrorResponse.class))),
+            @ApiResponse(responseCode = "404", description = "제안서 아이디가 존재하지 않습니다.", content = @Content(schema = @Schema(implementation = ErrorResponse.class))),
+            @ApiResponse(responseCode = "500", description = "서버 내부 오류 입니다.", content = @Content(schema = @Schema(implementation = ErrorResponse.class))),
+    })
+    @GetMapping("/offer")
```

Comment:
```suggestion
    @GetMapping("/offer/{offerId}")
```
p2
이거 작성하지 않아도 되나요..?

Reply:
쿼리로 작성했었네여..! 변경했습니다